lib/modules/platform/github/massage-markdown-links.ts

```diff
@@ -57,12 +67,18 @@ export function massageMarkdownLinks(content: string): string {
   try {
     const rightSpaces = content.replace(content.trimEnd(), '');
     const matches: UrlMatch[] = [];
-    remark().use(collectLinkPosition(content, matches)).processSync(content);
+    const newContent = content.replace(
+      regEx(/@&#8203;(?<name>[a-zA-Z1-9\-_.]*)/gi),
```

Comment:
```suggestion
      regEx(/@&#8203;(?<name>[a-z1-9\-_.]*)/gi),
```
simplify if case insensitive flag is used.